src/ecs/commands/runCommandInContainer.ts

```diff
@@ -109,14 +114,29 @@ export async function runCommandInContainer(
         })
 
         result = 'Succeeded'
-    } catch (error) {
-        if (CancellationError.isUserCancelled(error)) {
+    } catch (err) {
+        const error = err as Error
+        if (CancellationError.isUserCancelled(err)) {
             return
         }
 
         result = 'Failed'
-        getLogger().error('ecs: Failed to execute command in container, %O', error)
-        showViewLogsMessage(localize('AWS.ecs.runCommandInContainer.error', 'Failed to execute command in container.'))
+        const failedMessage = localize('AWS.ecs.runCommandInContainer.error', 'Failed to execute command in container.')
+        const missingPermissions = localize(
+            'AWS.ecs.runCommandInContainer.missingPermissions',
+            'Execute Command not allowed. See documentation for required permissions.'
+        )
+        const viewDocs = localize('AWS.generic.viewDocs', 'View Documentation')
+        getLogger().error('ecs: Failed to execute command in container, %O', err)
+        if (error.name === 'AccessDeniedException') {
```

Comment:
Just curious, but what's the `code` for this error (if any)?

Reply:
They were, at one point, different in v3 (which we are not using here ofc). May have been a bug.